tests/e2e/plot_utils.py

```diff
@@ -122,3 +123,11 @@ def generate_plotly_figure():
     plotly_fig = px.histogram(df, x="total_bill", y="tip", color="sex", marginal="rug", hover_data=df.columns)
 
     return plotly_fig
+
+
+def generate_seaborn_figure():
+    sample_size = 30
+    x = np.random.rand(sample_size) * 2 * np.pi
+    data = {"x": x, "y": np.sin(x), "c": np.random.randint(0, 2, sample_size), "arch": x > np.pi}
+    seaborn_fig = sns.relplot(data, x="x", y="y", hue="c", col="arch")
```

Comment:
This doesn't seem to work with `seaborn` lower than `0.12.0`. 
I get `ValueError: Could not interpret value 'y' for parameter 'y'` for `0.11.2`
Could you check this and if so, fix the seaborn version in `pyproject.toml` to `>=0.12.0` ?